test/integration/targets/plugin_loader/override/legacy.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: localhost
+  gather_facts: no
+  tasks:
+  - ping:
+    register: legacy
+  - ansible.builtin.ping:
+    register: builtin
+  - assert:
+      that:
+        - legacy['ping'] == 'legacy'
+        - builtin['ping'] == 'pong'
```

Comment:
This is a test case for https://github.com/ansible/ansible/pull/78814/files#diff-5675e463b6ce1fbe274e5e7453f83cd71e61091ea211513c93e7c0b4d527d637R1126

Reply:
This is now the only change in the PR.